### PR TITLE
deleted  $this->complete(); from __destruct

### PR DESCRIPTION
**Pull request by sergey-telpuk**

### CODE REVIEW

**Comment by sergey-telpuk** [comment]
#8

File: src/Task/ReceivedTask.php [hunk 1]
```diff
@@ -69,15 +69,7 @@ public function __construct(
      */
     public function __destruct()
```
[the remark this answers is not in the document]
Comment (sergey-telpuk) [reply src/Task/ReceivedTask.php:70]: you mean it's redundant, need to be deleted?

[the remark this answers is not in the document]
Comment (butschster) [reply src/Task/ReceivedTask.php:70]: Yes

[the remark this answers is not in the document]
Comment (sergey-telpuk) [reply src/Task/ReceivedTask.php]: @butschster could provide the release tag for it,  pls. I want to update my deps

**Pull request merged**